src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,84 @@
+@use "../../variables" as *;
+
+.footer {
+  text-align: center;
+  align-items: center;
+  color: $black;
+  font-size: $fontsize-s;
+
+  .contact {
+    border-bottom: 1px solid $gray_nobel;
+    padding: 52px 20px;
+    @media screen and(max-width:$breakpoint-md) {
+      padding: 32px 20px;
+    }
+    .wrapper {
+      padding-bottom: 20px;
+      img {
+        width: 150px;
+      }
+      adress {
+        line-height: 28px;
+      }
+    }
+    .icon-list {
+      display: flex;
+      justify-content: center;
+      gap: 14px;
+      .icon {
+        color: $black;
+
+        i {
+          width: 14px;
+          height: 14px;
+          transition: color 0.2s ease;
+          &:hover {
+            color: $gray_nobel;
+          }
+        }
+      }
+    }
+  }
+  nav {
+    padding: 16px;
+    background-color: $white_smoke;
+    ul {
+      margin-bottom: 0px;
+      padding: 0px;
+      list-style: none;
+      columns: 8;
+      column-rule: 1px solid;
+      column-gap: 8px;
+      @media screen and(max-width: $breakpoint-md) {
+        columns: 2;
+        column-rule: none;
+      }
+
+      li {
+        @media screen and(max-width: $breakpoint-md) {
+          text-align: start;
+          padding-bottom: 8px;
+        }
+      }
+      :nth-child(4n) {
+        padding-bottom: 0px;
+      }
+      a {
+        text-decoration: none;
+        color: inherit;
+        transition: background-color 0.2s ease;
+        &:hover {
+          color: $orange_bistre;
+        }
+      }
+    }
+  }
+  .copyright {
+    background-color: $black;
+    color: $white;
+    font-size: $fontsize-xs;
+    padding: 16px;
+    height: 44px;
```

```suggestion
    line-height: 1;
```
こうすることで、heightを固定しなくてもfigmaと同じ見た目になります。
デフォルトでは line-height がついてしまっているを、figmaの方に合わせfont-sizeと同一にしています。